### PR TITLE
adding play/pause toggle (temporary image)

### PR DESCRIPTION
**Pull request by rduivenvoorde**
First step to a play/pause button. The image is a quick fix, just wanted to show, and ask if interested.

See https://github.com/anitagraser/TimeManager/issues/175 in which I ask nirvn to make svg icons. If not I can make them myself...

### CODE REVIEW

**Comment by carolinux** [comment]
LGTM
I assume @anitagraser has no objections based on the discussion so far, so merging.

File: resources.qrc [hunk 1]
```diff
@@ -1,9 +1,11 @@
 <RCC>
   <qresource prefix="/">
+    <file>images/pause.png</file>
     <file>images/power_off.png</file>
     <file>images/back.png</file>
     <file>images/forward.png</file>
     <file>images/play.png</file>
```
Comment (carolinux) [inline resources.qrc]: why is it twice here?

**Comment by anitagraser** [comment]
Thank you!

File: resources.qrc (hunk 1 above)
Comment (rduivenvoorde) [inline resources.qrc]: Oops sorry, apparently both added by hand and via designer... should only be once...
Will remove it and squash to one commit. Hopefully adding svg images to this PR too

**Comment by carolinux** [comment]
Status: Awaiting svg's I suppose

**Comment by rduivenvoorde** [comment]
Let's just merge this, and hope Mathieu will create a nicer set of icons ...

**Pull request merged**